shovel/runner.py

```diff
@@ -69,6 +69,10 @@ def run(*args):
         if os.path.exists(path):  # pragma: no cover
             shovel.read(path, os.path.expanduser('~/'))
 
+    shovel_home = os.environ.get('SHOVEL_HOME', None)
```

Comment:
Technically, you don't need to pass `None` to `os.environ.get` since that's the default value returned.